### PR TITLE
Rename sprint label to in-progress (issue #701)

### PR DESCRIPTION
**Pull request by kalawac**
### Description

Resolves issue #701 by renaming `sprint` label to `in-progress` and removing reference to sprint in description. Also includes updates to `readme` and `How We Use GitHub` documents to remove references to sprints.

Tasks:

- [x] Global label config updated (`global.yml`)
- [x] `README.md` updated
- [x] `HOW_WE_USE_GITHUB.md` updated
- [x] `Kanban` project board name updated to `In Progress`

### CODE REVIEW

File: .github/global.yml [hunk 1]
```diff
@@ -38,8 +38,8 @@
   description: issue has been triaged but has not been earmarked for any upcoming release
   color: "ededed"
   aliases: [1_Backlog_Long_Term]
-- name: sprint
-  description: issue has been pulled into current sprint and is actively being worked on
+- name: in-progress
+  description: issue is actively being worked on
   color: "ededed"
```
Comment (kenodegard) [inline .github/global.yml:43]: We also want to add "sprint" as an alias so the old label will be automatically renamed for us

**Review by kenodegard — CHANGES_REQUESTED** [review]

**Review by kenodegard — CHANGES_REQUESTED** [review]
There are also some remaining references to "sprint" in the `HOW_WE_USE_GITHUB.md` document, can you take a glance there and update as needed?

**Comment by kalawac** [comment]
> There are also some remaining references to "sprint" in the `HOW_WE_USE_GITHUB.md` document, can you take a glance there and update as needed?

Updated. The name of the Kanban board will need to be updated to be in line with the updated document, but I don't have permissions.

**Comment by kenodegard** [comment]
> > There are also some remaining references to "sprint" in the `HOW_WE_USE_GITHUB.md` document, can you take a glance there and update as needed?
> 
> Updated. The name of the Kanban board will need to be updated to be in line with the updated document, but I don't have permissions.

Done ✅

**Review by jezdez — APPROVED** [review]

**Review by kenodegard — APPROVED** [review]

**Pull request merged**